.tekton/fedora-tier-1-eln-pull-request.yaml

```
@@ -7,7 +7,7 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch && "base-images/**".pathChanged()
```

```suggestion
    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main"&& "base-images/**".pathChanged()
```

and maybe remove line 11